sprimber-cucumber/src/main/java/com/griddynamics/qa/sprimber/discovery/CucumberTestBinder.java

```diff
@@ -114,6 +109,11 @@ private void fillPostConditions(String style, Node containerNode) {
                 .forEach(containerNode::addAfter);
     }
 
+    private void fillStepBeforeAndAfter(Node stepContainerNode) {
+        fillPreConditions("Before Step", stepContainerNode);
```

Comment:
Why do you have similar methods like fillStepBeforeAndAfter, fillFeatureHooks etc splitted by different classes?